### PR TITLE
private ann

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**

### CODE REVIEW

**Review by XiaoYhun — APPROVED** [review]

File: src/constants/env.ts [hunk 1]
```diff
@@ -29,15 +29,17 @@ export const MIXPANEL_PROJECT_TOKEN = required('MIXPANEL_PROJECT_TOKEN')
...
+//   ? ENV_TYPE.LOCAL
+//   : process.env.REACT_APP_TAG.startsWith('adpr')
+//   ? ENV_TYPE.ADPR
+//   : process.env.REACT_APP_TAG.startsWith('main-stg')
+//   ? ENV_TYPE.STG
+//   : process.env.REACT_APP_TAG.startsWith('main')
+//   ? ENV_TYPE.DEV
+//   : ENV_TYPE.PROD
+
+export const ENV_LEVEL = (() => ENV_TYPE.STG)()
```
Comment (nhd98z) [inline src/constants/env.ts:42]: Revert this before merge.

File: src/components/Announcement/type.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export enum PrivateAnnouncementType {
   TRENDING_SOON_TOKEN = 'TRENDING_SOON',
   POOL_POSITION = 'ELASTIC_POOLS',
   PRICE_ALERT = 'PRICE_ALERT',
+  PRIVATE_MESSAGE = 'DIRECT_MESSAGE', // for some specific addresses
```
Comment (hungdoansy) [inline src/components/Announcement/type.ts:19]: 2 different texts?

File: src/pages/NotificationCenter/PriceAlerts/index.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ const PriceAlerts = () => {
     setDisabledClearAll(false)
   }
 
-  useAckAnnouncement(getAnnouncementsTemplateIds().PRICE_ALERT)
+  useAckAnnouncement(getAnnouncementsTemplateIds()?.PRICE_ALERT)
```
Comment (nhd98z) [inline src/pages/NotificationCenter/PriceAlerts/index.tsx:62]: Why need ? here?

**Review by nhd98z — DISMISSED** [review]

File: src/pages/NotificationCenter/PriceAlerts/index.tsx (hunk 3 above)
Comment (nguyenhoaidanh) [reply src/pages/NotificationCenter/PriceAlerts/index.tsx:62]: yep, no need.

File: src/constants/env.ts (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/constants/env.ts:42]: okay

**Review by viet-nv — APPROVED** [review]

**Pull request merged**